homework/homework_section_2/javascript/app.js

```diff
@@ -0,0 +1,107 @@
+console.log("Part 1: Terms");
+
+//What is the difference between interpolation and concatenation? Give an example of each.
+console.log("1. Interpolation vs Concatenation. Interpolation is the process of linking strings/variables/constants together. Concatenation is the process of appending strings together.")
+//Interpolation
+//==> Process of evaluating an expression, specifically a string literal, and replacing variable placeholders by their values. 
+//==> MUST BE WITH BACK-TICKS!
+var interpolationExample = 'interpolation';
+console.log(`This is an example of ${interpolationExample}.`);
+//Concatenation
+var concatenationExample = 'concate' + 'nation';
+console.log(`This is an example of ${concatenationExample}.`);
+
```

Comment:
good job -- put more simply, interpolation is including a variable containing a string in a string literal. concatenation is "adding strings together"